src/main/java/tv/banko/tutorialreloaded/utils/Config.java

```diff
@@ -12,21 +12,17 @@
 
```

Comment:
Why not replace the whole class with `Plugin#getConfig()` wouldn't make a difference